XenModel/Actions/Updates/DownloadUpdatesXmlAction.cs

```diff
@@ -83,10 +83,10 @@ public DownloadUpdatesXmlAction(bool checkForXenCenter, bool checkForServerVersi
             _checkForXenCenter = checkForXenCenter;
             _checkForServerVersion = checkForServerVersion;
             _checkForPatches = checkForPatches;
-            _checkForUpdatesUrl = string.IsNullOrEmpty(checkForUpdatesUrl) ? InvisibleMessages.XENSERVER_UPDATE_URL : checkForUpdatesUrl;
+            _checkForUpdatesUrl = checkForUpdatesUrl;

```

Comment:
The `IsNullOrEmpty `check is still needed for the case when this constructor is called with the last parameter null, like this usage:
`this(checkForXenCenter, checkForServerVersion, checkForPatches, null)`
on line 90 below.